### ISSUE TITLE
[BUG] "Last change" and "last update" popup entries not updated.

### ISSUE THREAD

**Issue by mcv93us** [ExperienceLovelace/ha-floorplan#207]
**Describe the bug**
In Ha-Floorplan, when I hover the mouse over an entity, a popup appears containing the entity name, status and other information including the last change and the latest update, and it is these two items that I always find written " 3 weeks ago "even though the entity has just been updated.

**To Reproduce**
1. In Ha-Floorplan, move the mouse over any entity and a popup will appear.'
2. Note that the entries "last change" and "last update" have incorrect dates.

**Expected behavior**
I expect to find consistent dates, if a sensor has just activated I will need to have updated the date of the last change exactly as its status is updated.

**Screenshots**
I am attaching a screenshot of the popup in question and another screenshot of the developer screen where you can see the same sensor but with the correct "last change" and "last update" data.

**Desktop (please complete the following information):**
 - OS: Windows 10 Home 21H2 (build 19044.1566)
 - Browser Chrome
 - Version 98.0.4758.102

**Smartphone (please complete the following information):**
 - Device: Xiaomi Redmi Note 4
 - OS: Android 7.0 NRD90M
 - Browser Chrome
 - Version 98.0.4758.101

![Immagine2](https://user-images.githubusercontent.com/9490541/156269477-ed14132b-2945-4720-8677-2ba2f845a628.png)

![Immagine1](https://user-images.githubusercontent.com/9490541/156269460-262bbfd9-3d30-4a04-ae3a-23a3c14c5674.png)

**Additional context**
none

**Comment by OzGav** [comment]
Hi. I just checked my floorplan and I don't see this problem. Is it all your entities or just this one?

**Comment by mcv93us** [comment]
They are all entities on my floorplan.
The version in use is the latest 1.0.30 and HA is also updated to the latest version 2022.2.9.
If it can be the installation is supervised on docker version 210.10.12 with s.o. Raspbian Buster.

**Comment by mcv93us** [comment]
After updating HA to version 2022.3.0 I saw that in Ha-Floorplan the "last change" and "last update" attributes now work again, reporting the exact date and time of the last change.
I close this issue.

**Comment by mcv93us** [comment]
I spoke too soon.
A little while ago after updating HA to version 2022.3.1 the problem reappeared.
From the changelog it didn't mention anything about entity changes so I don't know where the catch is.
I re-issue this problem in the hope of a solution.

**Comment by OzGav** [comment]
I just had a look at mine again and I might see what you are referring to. I looked at a light and it was saying last changed days ago yet I just saw it go from off to on. However, that light turned on and off from an automation. When I touched the floorplan to turn it on the last update attribute in floorplan said seconds ago. So I am not sure if this is by design or not…. Let me play with it some more over the next few days…

**Comment by yangqian** [comment]
I notice that it is working properly in Chrome but not firefox 98.0.

**Comment by OzGav** [comment]
Yeah I have looked again and I think mine is working ok. @mcv93us are you certain the entity in HA and the entity you are looking at in developer tools is Identical?

**Comment by mcv93us** [comment]
Yes, I'm also sure because it does it to me with all entities: door sensors, PIRs, switches, fans, IP cameras, etc ...
I also tried with Firefox but nothing changes.

**Comment by OzGav** [comment]
Yeah I'm sorry as I have three different floorplans and those attributes work fine for me. You will need to do some more testing to try and track down where the issue is. Maybe create a new floorplan with just one entity (like just a blank image with a black square in the middle) and see if there is a problem with that one?

**Comment by mcv93us** [comment]
I have created a new floorplan, the simplest that can be done: a square that acts as a switch for a light but has not changed anything, I still have the items "last change" and "last update" not updated.
Since I did the update of HA to version 2022.3.5 the last change and the last update in Floorplan are "just now" on any entity present.
If I am the only one with this problem then let's leave it alone, after all it is useless to go crazy for this problem.
Thanks again for your patience and time spent!

**Comment by OzGav** [comment]
Yeah fair enough. As I said I can't reproduce this. If you find more info in the future you can reopen or open a new issue. Cheers
